pkg/pxc/backup/restore.go

```diff
@@ -40,6 +40,12 @@ func PVCRestoreService(cr *api.PerconaXtraDBClusterRestore) *corev1.Service {
 			},
 		},
 	}
+
+	if cr.Annotations["headless-service"] == "true" {
```

Comment:
@spron-in what do you think about this annotation? it'll look like:

```
apiVersion: pxc.percona.com/v1
kind: PerconaXtraDBClusterRestore
metadata:
  name: restore1
  annotations:
    headless-service: true
spec:
  pxcCluster: cluster1
  backupName: backup1
```

Reply:
To avoid confusion shall we make it `percona.com/headless-service: true` ? Just to make sure it is clear that it is Percona's annotation and to align with our existing standard for labels.

Reply:
@zulh-civo can you add `percona.com/` prefix to the annotation?

Reply:
Sorry for the delay. I've updated the annotation key.